augur/filter/_run.py

```diff
@@ -475,6 +475,9 @@ def run(args):
         include_exclude_rules.force_include_strains.__name__: "{count} strains were added back because they were in {include_file}",
         include_exclude_rules.force_include_where.__name__: "{count} sequences were added back because of '{include_where}'",
     }
+
+    # Sort for reproducible output.
+    filter_counts = dict(sorted(filter_counts.items()))
```

Comment:
Hmm. The order of filtering can sometimes matter, i.e. it's not always commutative like addition, right? I don't know if we have filters like that, but it seems like the output of counts filtered by X might matter and not be arbitrary?

Reply:
Yeah, good point. Even though users don't control the order of filtering, at least we should make the report output reflect the predefined order.

Dropped 0e9086fbf575baa6a83400addfb780b8310fbfaf in [force-push](https://github.com/nextstrain/augur/compare/e37c32f65c7f922b5d9dbda6e4c3f4c1a5f7c4d6..3f8a2842b6a0d1cc3640e231ba9b1187084732cf).